pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -95,6 +95,59 @@ type ClusterList struct {
 	Items           []Cluster `json:"items"`
 }
 
+// ConditionStatus indicates the status of a condition (true, false, or unknown).
+type ConditionStatus string
```

Comment:
The code here is duplicating the [`corev1` API](https://github.com/Azure/ARO-RP/blob/master/vendor/k8s.io/api/core/v1/types.go#L2393) or operator related https://github.com/Azure/ARO-RP/blob/d8d2303a9a7f1074ff87dde9d04d701cd0e3ef69/vendor/github.com/openshift/api/operator/v1/types.go#L166